tests/llm/conftest.py

```
@@ -221,45 +242,172 @@ def force_pytest_output(request):
-            provider_msg = (
-                f"Tried to use AzureAI (model: {classifier_model}) because AZURE_API_BASE was set. "
-                "Check AZURE_API_BASE, AZURE_API_KEY, AZURE_API_VERSION, or unset them to use OpenAI."
-            )
+            provider_msg = f"Tried to use Azure for classifier (model: {classifier_model}). Check AZURE_API_BASE, AZURE_API_KEY, AZURE_API_VERSION, or unset AZURE_API_BASE to use OpenAI."
         else:
-            provider_msg = (
-                f"Tried to use OpenAI (model: {classifier_model}). "
-                "Check OPENAI_API_KEY or set AZURE_API_BASE to use Azure AI."
-            )
+            provider_msg = f"Tried to use OpenAI for classifier (model: {classifier_model}). Check OPENAI_API_KEY or set AZURE_API_BASE to use Azure."
+        error_messages.append(f"{provider_msg}\n    Error: {str(e)}")
```

Ruff (0.12.2)

Use explicit conversion flag

Replace with conversion flag

(RUF010)